DscResources/MSFT_xMySqlServer/MSFT_xMySqlServer.schema.mof

```diff
@@ -2,19 +2,10 @@
 [ClassVersion("1.0.0"),FriendlyName("xMySqlServer")] 
 class MSFT_xMySqlServer : OMI_BaseResource
 {
-  [Key, Description("Provides the service name to use during setup of MySQL")]
-  String ServiceName;
-  
-  [Write,
-  ValueMap{"Present", "Absent"},Values{"Present", "Absent"},
-  Description("Ensure resource is present or absent")]
-  String Ensure;
-  
-  [required,
-  Write,
-  EmbeddedInstance("MSFT_Credential"),
-  Description("The root credential that is used to install mySql server.")]
-  String RootPassword;
+    [Key, Description("mySql Version Number")] String MySqlVersion;
```

Comment:
I agree that adding version support is important , but this change has a solution for it from a wrong angle I believe. Original version of the resource was using ServiceName as a key property as on the same machine you can run multiple MySQL servers of the same version if you install them as services using unique names.